JSONString.java

```diff
@@ -33,21 +33,27 @@ public JSONString(String value) {
    * Convert to a string (e.g., for printing).
    */
   public String toString() {
-    return "";          // STUB
+    return (this.value == null)
+      ? "null"
+      : this.value.toString();
   } // toString()
 
   /**
    * Compare to another object.
    */
   public boolean equals(Object other) {
-    return true;        // STUB
+    return ((this == other)
+            || ((other instanceof JSONString)
+                && (this.value.equals(((JSONString) other).value))));
   } // equals(Object)
 
   /**
    * Compute the hash code.
    */
   public int hashCode() {
-    return 0;           // STUB
+    return (this.value == null)
```

Comment:
Again, is `JSONString` nullable?

Reply:
Like above, it'll never be null, but its consistent